settings.gradle

```diff
@@ -15,5 +15,6 @@ include "eventuate-local-java-quarkus-jdbc-tests-common"
 include "eventuate-local-java-quarkus-jdbc-tests"
 include "eventuate-local-java-quarkus-jdbc-snapshot-tests"
 include "eventuate-local-java-quarkus-jdbc-metadata-tests"
+include "eventuate-local-quarkus-bom"
```

Comment:
Where are the files?

Reply:
@cer what files? bom folders contain only build directory that ignored by git. Also git ignores empty folders.

Reply:
Oh. I'd forgotten :-)